src/BEditaClient.php

```diff
@@ -357,6 +357,53 @@ public function remove($id) : ?array
         return $this->delete(sprintf('/trash/%s', $id));
     }
 
+    /**
+     * Upload file (POST)
+     *
+     * @param string $filename The file name
+     * @param string $filepath The file full path
```

Comment:
I'd say - `string $filepath File full path: could be on a local filesystem or a remote reachable URL`